flaml/tune/tune.py

```diff
@@ -511,7 +492,11 @@ def easy_objective(config):
                 logger.warning("Using CFO for search. To use BlendSearch, run: pip install flaml[blendsearch]")
             metric = metric or DEFAULT_METRIC
         else:
-            SearchAlgorithm = CFO
+            assert "lexico_algorithm" in lexico_objectives and lexico_objectives["lexico_algorithm"] in [
+                "CFO",
+                "BlendSearch",
+            ], 'When performing lexicographic optimization, "lexico_algorithm" should be provided in lexicographic objectives (CFO or BlendSearch).'
+            SearchAlgorithm = CFO if lexico_objectives["lexico_algorithm"] == "CFO" else BlendSearch

```

Comment:
I think it is simpler for users to specify algorithms from lexico_objectives. Because `search_alg`  in tune represents algorithm instance rather than strings. We have also not tried passing instances of LexiFlow to use tune before.

Reply:
You can support str type `search_alg` in `tune` to make it easier. But don't create a new argument which is confusing.
`AutoML` uses `tune` by passing instance of CFO.